root/static/scripts/relationship-editor/common/fields.js

```diff
@@ -541,6 +541,8 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
             if (entity === entities[1]) {
```

Comment:
Same question as **B**

Reply:
Same here, I'd say it's an error if this function ever receives an entity not in the `entities` array.